### PR TITLE
LPD-25627 | Show Object name instead of Table name for elements in data migration center

### PR DESCRIPTION
**Pull request by jaimelr10**
Description: Previously, we showed the table name (eg. `C_Students`). Now we want to show the Object name (eg. `Students`). Furthermore, we want to show the version with a format `v1.0` instead of the current one: `v1_0`.

Screenshots:
image
image

Jira Ticket: https://liferay.atlassian.net/browse/LPD-25627

### CODE REVIEW

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by pyoo47** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
It fails locally too:

```
     [exec] tests/batch-planner/export.spec.ts(380,19): error TS2339: Property 'object' does not exist on type 'ApiHelpers'.
     [exec] tests/batch-planner/import.spec.ts(1267,19): error TS2339: Property 'object' does not exist on type 'ApiHelpers'.
```

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java [hunk 1]
```diff
@@ -215,6 +215,11 @@ public ObjectEntry getObjectEntry(Long objectEntryId) throws Exception {
 			objectEntryId);
 	}
 
+	@Override
+	public String getResourceName() {
+		return _objectDefinition.getShortName();
```
Comment (ccorreagg) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java:220]: In the commit message: "overided" :thinking:

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
Included an extra commit and rebased with master

**Review by ccorreagg — COMMENTED** [review]
As we talked, depending on the PM decission, we would update the PR with the changes in the version calculation. I included a comment because a commit message is incorrect, the rest of the PR is great!

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
Rebased with master

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by jaimelr10** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by 4lejandrito** [comment]
ci:forward:force

**Pull request closed without merging**